static/styles/settings.css

```diff
@@ -569,9 +573,28 @@ input[type="checkbox"] {
     margin-top: 10px;
 }
 
-.language_selection_widget .language_selection_button {
-    text-decoration: none;
-    margin-left: 20px;
+.language_selection_widget {
+    margin-bottom: 20px;
+
+    .title {
+        margin-bottom: 5px;
+    }
+
+    .language_selection_button {
+        text-decoration: none;
+        color: hsl(0, 0%, 20%);
+    }
+}
+
+.realm_default_language {
+    .input-group {
+        display: inline;
```

Comment:
This CSS is actually not applied since there is no `input-group` element inside `realm_default_language` element.

Reply:
Looks like I forgot to remove the css I added for the other direction was heading with this.